packages/vue/src/components/atoms/SfQuantitySelector/SfQuantitySelector.stories.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { storiesOf } from "@storybook/vue";
+import { withKnobs, number } from "@storybook/addon-knobs";
+
+import SfQuantitySelector from "./SfQuantitySelector.vue";
+
+storiesOf("Atoms|QuantitySelector", module)
+  .addDecorator(withKnobs)
+  .add("Common", () => ({
+    components: { SfQuantitySelector },
+    data() {
+      return {
+        value: 1
+      };
+    },
+    props: {
+      stock: {
+        default: number("stock", 99, {}, "Props")
+      },
+      min: {
+        default: number("min (props)", 1, {}, "Propss")
```

Comment:
Please use the new convention, so remove ` (props)` and remove the double-`s` in `"Propss"` 😉

Reply:
fixed: https://github.com/DivanteLtd/storefront-ui/pull/485/commits/70e2c150426687f9d691732144c9997606aade7a